pkgs/by-name/op/openhue-cli/package.nix

```diff
@@ -14,22 +14,29 @@ buildGoModule (finalAttrs: {
     owner = "openhue";
     repo = "openhue-cli";
     tag = finalAttrs.version;
-    hash = "sha256-yIPqjKIvYk2Y9BYieqrm5QvvAHnuImXEDbI1JOy0kBA=";
+    hash = "sha256-LSaHE3gdjpNea6o+D/JGvHtwvG13LbHv2pDcZhlIoEE=";
+    leaveDotGit = true;
+    postFetch = ''
+      cd "$out"
+      git rev-parse HEAD > $out/COMMIT
```

Comment:
If this isn't deemed too hacky, I'll leave it in, otherwise I'll set/leave rev to "undefined"